app/src/main/shared/services/AbstractLocalService.ts

```diff
@@ -208,8 +208,10 @@ export abstract class AbstractLocalService {
     }
 
     notifyOnErrors(error: any) {
-        this.initOptions();
-        notificationActionCreators.notify(error.toString(), "error")(this.dispatch, null);
+        if (this.options.notificationOnError) {
```

Comment:
I made this change so that we wouldn't see pop up errors every five seconds when polling for run status if there was a problem contacting the API. But I'm a bit confused by the previous behaviour of the base class. I would have thought that setting notificationOnError in the options would prevent notification, not sure why it was calling initOptions before?

Reply:
Not sure either tbh. We should probably put in some work on the error handling at some point, it could do with a refactor and redesign, both the code and the user experience are a bit confusing at the moment!